config/helm/chart/default/tests/Common/csi/role-csi_test.yaml

```diff
@@ -47,18 +47,20 @@ tests:
                 - ""
               resources:
                 - secrets
+                - configmaps
```

Comment:
do I understand that correctly that you moved configmpas here because it has the same permissions like secrets?

Reply:
Yes, it is unnecessary to write two sections with the same permissions